Cargo.toml

```diff
@@ -40,6 +40,7 @@ fnv = "1.0"
 lazy_static = "0.2"
 libc = {version = "0.2", optional = true}
 cfg-if = "0.1"
+spin = "0.4"
```

Comment:
I think you may need to disable the `default` [0] feature.

[0] https://github.com/mvdnes/spin-rs/blob/abf8de701d7c2e81c7a98b191edf3133577ea499/Cargo.toml#L24